cms/templates/product_page.html

```diff
@@ -5,6 +5,7 @@
 {% block title %}{{ site_name }} | {{ page.title }}{% endblock %}
 
 {% block content %}
+    {% include "top_nav.html" %}
```

Comment:
It is not be the part of base.html  ?

Reply:
I'd rather keep it separate because we might need to migrate that over to React in chunks.